scripts/install-runner

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -e
+
+latest_runner_version=$(curl -I -v -s https://github.com/actions/runner/releases/latest 2>&1 | perl -ne 'next unless s/^< location: //; s{.*/v}{}; s/\s+//; print')
+
+if [ -e /etc/image_runner_version ]; then
+  image_runner_version=$(cat /etc/image_runner_version)
+  if [ "$image_runner_version" = "$latest_runner_version" ]; then
+    exit 0
+  fi
+  echo "Current runner version ($image_runner_version) is not up to date."
+fi
+
+echo "Installing latest version ($latest_runner_version)"
+curl -sL "https://github.com/actions/runner/releases/download/v${latest_runner_version}/actions-runner-linux-x64-${latest_runner_version}.tar.gz" | tar xzvC /home/runner && \
+    /home/runner/bin/installdependencies.sh && \
```

Comment:
can we make this run only the first time, it doesn't need to run everytime, which would pull apt-indexes etc.

Reply:
Are you sure about this?

Isn't it possible that a newer runner adds a new system library as a dependency? Or requires a newer version of a system library?

We'll do whatever it takes to get a version of this PR merged, but, I'm not sure that changing this would be technically correct.

Reply:
I'd say the chances of something breaking is slim. You can make it opt-in via an environment-variable, but have it off by default.

Reply:
Good, that's the current state of this PR.

Anything else you need?